src/main/java/com/firebolt/jdbc/connection/settings/FireboltSessionProperty.java

```diff
@@ -44,17 +44,18 @@ public enum FireboltSessionProperty {
 				 */
 			"compress", true, Boolean.class, "Whether to compress transferred data or not. Compressed by default"),
 	DATABASE("database", null, String.class, "default database name"),
-	PASSWORD("password", null, String.class, "user password - null by default"),
-	USER("user", null, String.class, "user name - null by default"),
+	CLIENT_SECRET("client_secret", null, String.class, "user password - null by default", "password"),
+	CLIENT_ID("client_id", null, String.class, "user name - null by default", "user"),
 	HOST("host", null, String.class, "Firebolt host - null by default"),
 	PORT("port", null, Integer.class, "Firebolt port - null by default"),
 	ENGINE("engine", null, String.class, "engine - null by default", "engine_name"),
-	ACCOUNT("account", null, String.class, "account - null by default"),
+	ACCOUNT("account", "firebolt", String.class, "account - null by default"),
+	ACCOUNT_ID("account_id", "firebolt", String.class, "accountId - null by default"),
 	LOG_RESULT_SET("log_result_set", false, Boolean.class,
 			"When set to true, the result of the queries executed are logged with the log level INFO. This has a negative performance impact and should be enabled only for debugging purposes"),
 	USER_DRIVERS("user_drivers", null, String.class, "user drivers"),
 	USER_CLIENTS("user_clients", null, String.class, "user clients"),
-
+	ENVIRONMENT("environment", "app", String.class, "Firebolt environment", "env"),
```

Comment:
We should be able to determine this value from the `HOST`, right? e.g. api.app.firebolt.io -> env=app

Reply:
Hopefully fixed. Please take a look on `FireboltPropertiesTest.hostAndEnvironment()` and `FireboltProperties.getEnvironment()`

Reply:
Yeah, looks good. Having both environment and host is somewhat redundant, since we duplicate the info, but as long as it makes the interaction with the driver easier for us, why not.